bundle/edu.gemini.model.p1/src/test/scala/edu/gemini/model/p1/dtree/RootSpec.scala

```diff
@@ -12,8 +12,8 @@ class RootSpec extends SpecificationWithJUnit {
       root.choices must contain(Instrument.Gsaoi)
     }
     "include Texes" in {
-      val root = new Root(Semester(2015, A))
-      root.choices must not contain (Instrument.Texes)
+      val root = new Root(Semester(2016, A))
```

Comment:
That's right, these test may not make too much sense these days as the way to include/hide instruments has changed, but for completness they should all use 2016A